libraries/RC_Channel/RC_Channel.h

```diff
@@ -358,6 +358,11 @@ class RC_Channels {
         return get_singleton() != nullptr && (_options & uint32_t(Option::FPORT_PAD));
     }
 
+    // should a channel reverse option affect aux switches
+    bool switch_reverse_allowed(void) const {
+        return get_singleton() != nullptr && (_options & uint32_t(Option::ALLOW_SWITCH_REV));
```

Comment:
i think we can remove the get_singleton() check on all these functions, it looks like it was leftover from when we needed singleton to get at _options